json-rpc/types/Quantity.md

```diff
@@ -0,0 +1,33 @@
+---
+title: JSPON RPC Quantity Type Spec
+author: Alita Moore (@alita-moore)
+discussions-to: https://github.com/ethereum-oasis/eth1.x-JSON-RPC-API-standard
+created: 2021-04-13
+---
+
+# `Quantity`
+
+##### Description
```

Comment:
Recommend against skipping heading levels, it can break some markdown tools like table of contents generators.